contents/docs/libraries/python/index.mdx

```diff
@@ -33,25 +33,18 @@ from posthog import Posthog
 posthog = Posthog('<ph_project_api_key>', host='<ph_instance_address>', personal_api_key='<ph_personal_api_key>')
 ```
 
+> **Note:** For versions older than `2.5.0`, you'll need to import the `Client` class from `posthog.client` instead.
```

Comment:
Maybe just keep it the old way where they don't need to initialize a client at all